packages/@okta/vuepress-site/guides/sign-into-mobile-app/check-for-tokens/android/checkfortoken.md

```diff
@@ -1 +1,56 @@
-check for token on startup code example
\ No newline at end of file
+After `AuthenticateClient` instance is created you can check if the user is logged in with the following:
```

Comment:
After the `AuthenticateClient` instance is created, make sure that the user is signed in: